Header header
uint32 seq
RangeBearing[] range_bearings